README.md

```diff
@@ -17,9 +17,9 @@ software architecture and example usage.
 ### Table of Contents
```

Comment:
you can remove `https://github.com/CorfuDB/CorfuDB/tree/cli_script_bugfixes#install-from-debian-package`

Reply:
It's probably okay to just mention that maven 8 is required and omit the java installation steps.

Reply:
You can remove these references as well 

```java

<repositories>
    <repository>
        <id>corfu-mvn-repo</id>
        <url>https://raw.github.com/CorfuDB/Corfu-Repos/mvn-repo/</url>
        <snapshots>
            <enabled>true</enabled>
            <updatePolicy>always</updatePolicy>
        </snapshots>
    </repository>
</repositories>

```

Reply:
@xcchang Where do we push the runtime artifacts?

This isn't correct, right?

```java

 <dependency>
    <groupId>org.corfudb</groupId>
    <artifactId>runtime</artifactId>
    <version>0.1-SNAPSHOT</version>
    <scope>compile</scope>
</dependency>
```

Reply:
Also, the example in https://github.com/CorfuDB/CorfuDB/tree/cli_script_bugfixes#now-i-want-to-write-a-program-that-uses-corfu doesn't make a lot of sense. It should print the value of the map.get before and after to demonstrate the db access.

Reply:
> Also, the example in https://github.com/CorfuDB/CorfuDB/tree/cli_script_bugfixes#now-i-want-to-write-a-program-that-uses-corfu doesn't make a lot of sense. It should print the value of the map.get before and after to demonstrate the db access.

I'm not sure what you mean by this change. The current code shows the db access across clients by printing out the value of the variable currently stored.

Reply:
I see. Just make sure the dependency is fixed. We don't push to `//raw.github.com/CorfuDB/Corfu-Repos/mvn-repo`.
@xcchang made this change, I think.

Reply:
> @xcchang Where do we push the runtime artifacts?
> 
> This isn't correct, right?
> 
> ```java
>  <dependency>
>     <groupId>org.corfudb</groupId>
>     <artifactId>runtime</artifactId>
>     <version>0.1-SNAPSHOT</version>
>     <scope>compile</scope>
> </dependency>
> ```

We are pushing to GitHub Packages now: https://github.com/orgs/CorfuDB/packages?repo_name=CorfuDB. No the version string should be `0.3.1-SNAPSHOT`.